prow/jobs/kyma/kyma-release-candidate.yaml

```diff
@@ -24,9 +24,9 @@ job_template: &job_template
           - name: KYMA_PROJECT_DIR
             value: "/home/prow/go/src/github.com/kyma-project"
           - name: CLOUDSDK_COMPUTE_ZONE
-            value: "europe-west3-c"
+            value: "europe-west1-c"
```

Comment:
Why not west4 here? (and below)

Reply:
The original idea was to have "release jobs" in different zone, like it was done previously.